lib-persistence-manager/persistencemgr/redis.go

```diff
@@ -90,14 +90,14 @@ func sentinelNewClient(dbConfig *Config) *redisSentinel.SentinelClient {
 //GetCurrentMasterHostPort is to get the current Redis Master IP and Port from Sentinel.
 func GetCurrentMasterHostPort(dbConfig *Config) (string, string) {
 	sentinelClient := sentinelNewClient(dbConfig)
+	log.Debug("sentinelClient in GetCurrentMasterHostPort : ", sentinelClient)
```

Comment:
print Addr as string and DB instance as int

Reply:
added MasterIP and masterport in logs to debug